package.json

```diff
@@ -23,7 +23,7 @@
     "sbc-common-components": "4.0.1",
     "uuid": "^9.0.0",
     "vue": "^3.3.4",
-    "vue-router": "^3.6.5",
+    "vue-router": "^4.2.5",
```

Comment:
Ketaki, Leo - Cameron was kind enough to test this component in his (ppr/mhr) Vue UI, but needed these changes. He's swamped so I said we'd take over these changes. Please review. Who has time to continue this PR?

Since this file has been updated, its app version should be updated, too.

Reply:
Will do!

Reply:
Cam, we can do that.

Reply:
Ok great, sounds good!

Reply:
Thank you so much! Really appreciate this.

Can you reuse your prev ticket number or do you want a new 2-pointer for this?

Reply:
No problem! I think we should create a new 2-pointer for this.

Reply:
Make it so :)

Reply:
Ticket for completing this PR: https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/18974